### ISSUE TITLE
Prince of Persia: The Two Thrones: Bloom issue

### ISSUE THREAD

**Issue by Triticum0** [xemu-project/xemu#873]
### Title

https://xemu.app/titles/55530058/#Prince-of-Persia-The-Two-Thrones

### Bug Description

When saving the game the screen goes white in the save menu making it impossible to save the game and completely cancelling the save does make the state irreversible making it impossible to progress.

### Expected Behavior

saving shouldn't brick the game

### xemu Version

0.6.2-79-ga32f271d55

### System Information

| Field | Value |
|-------|-------|
| OS			| Windows			| 
| CPU			|Intel(R) Core(TM) i5-10400F CPU @ 2.90GHz		| 
| Graphics Device	|NVIDIA GeForce GTX 970/PCIe/SSE2 			| 
| Graphics Driver	|  4.0.0 NVIDIA 511.79	| 

### Additional Context

This issue was from Alexwpi#1412 so no screenshots, @mborgerson tested it last so can take a look at the issue when it's convenient 

Regression: Playble -> Intro

**Comment by mborgerson** [comment]
Can you rephrase title to indicate this is a graphics issue

**Comment by Triticum0** [comment]
Seems to be fixed
